demos/common/cpp/Dockerfile

```diff
@@ -88,7 +88,7 @@ COPY .bazelrc WORKSPACE /build/
 COPY external /build/external/
 COPY third_party /build/third_party/
 
-RUN mkdir -p /opt/intel/openvino && cd /opt/intel/openvino && curl -s https://storage.openvinotoolkit.org/repositories/openvino/packages/2022.1/opencv/openvino_opencv_ubuntu20.tgz | tar -xzf - && cd /build
+RUN cp /build/third_party/opencv/install_opencv.sh /build/third_party/opencv/opencv_cmake_flags.txt . && ./install_opencv.sh
```

Comment:
Why do we have Dockerfile changes here in MR for metrics CLI config support?

Reply:
Added during rebase. Will be merged automatically to develop.